site/docs/selection/nearest.md

```diff
@@ -16,3 +16,7 @@ In the scatterplot below, points <select onchange="changeSpec('paintbrush_neares
 The `nearest` transform also respects any [spatial encoding projections](project.html) applied to the selection. For instance, in the example below, moving the mouse cursor back-and-forth snaps the vertical rule and label to the nearest `date` value.
 
 <div id="paintbrush_nearest" class="vl-example" data-name="stocks_nearest_index"></div>
+
+## Current Limitations
+
+* The `nearest` transform is not supported for continuous mark types (i.e., `line` and `area`). For these mark types, consider layering a discrete mark type (e.g., `point`) with a 0-value `opacity` as in the last example above.
```

Comment:
The example above doesn't use 0 opacity. How about "consider layering a discrete mark type (e.g., `point`) as in the last example above but use a 0-value `opacity`."

Reply:
Not sure what you mean -- [here](https://github.com/vega/vega-lite/pull/3009/files#diff-41b1e96b395ea6629b4c649c14778f7bR28) is the 0-value `opacity` on the `point` mark.

Reply:
Oh, I missed that update. 👼

Reply:
It's also not quite the right semantics but the best we can do. What I mean is that the closest line may not be the one that has the closest control point.